Sources/WalletConnectRouter/Router/Router.swift

```diff
@@ -0,0 +1,9 @@
+import UIKit
+
+public struct WalletConnectRouter {
+    public static func goBack(uri: String) {
+        DispatchQueue.main.async {
```

Comment:
do we nee separate package to call `UIApplication.shared.open`?
May be just remove Router package?